### ISSUE TITLE
pointerDown() throws exception on multitouch: Uncaught TypeError: Cannot read property '_element' of null

### ISSUE THREAD

**Issue by suit4** [taye/interact.js#301]
Without using gestures in my programming, I touched my big touchscreen with 5 fingers.

This led to the exception 
Uncaught TypeError: Cannot read property '_element' of null
from around here:
```
if (this.pointerIds.length > 1 && this.target._element === this.element) {
    var newAction = validateAction(forceAction || this.target.getAction(pointer, event, this, this.element), this.target);
    if (withinInteractionLimit(this.target, this.element, newAction)) {
        action = newAction;
    }
    this.prepared.name = null;
}
```

Under certain circumstances, this.target is undefined, hence this.target._element goes sideways.

interact.js version 1.2.6
tested in Chrome version 46.0.2490.71 m (64-bit)

**Comment by taye** [comment]
Can you try updating to the latest commit of the `unstable` branch? https://rawgit.com/taye/interact.js/unstable/dist/interact.js

**Comment by suit4** [comment]
Sure.
Any hint on how to build the standalone version?

**Comment by taye** [comment]
From the `master` branch, after `npm install` run

```shell
npm run gulp build
```

and the generated files should be in `dist/`. I've already built the files from the latest commit in `master` and commit to the `unstable` branch so you can use the files from there.

**Comment by suit4** [comment]
Thanks for the fix.
If I run ```npm run gulp build```, I do not get the single file / standalone interact.js, but a file interact.js, that still references in ./src/utils ...

Anyway, I found the part you modified in interactionFinder.js and merged in into the current 1.2.6.

Apparently, this fixes the bug.

Thanks!

**Comment by taye** [comment]
>If I run npm run gulp build, I do not get the single file / standalone interact.js, but a file interact.js, that still references in ./src/utils ...

The file is standalone but the bundled modules still `require` other modules as if they were in separate files. browserify resolves the `require` calls to different parts of the same file.

I've committed an improvement to the last fix.
